tests/plugins/test_pull_base_image.py

```diff
@@ -672,7 +679,11 @@ def workflow_callback(workflow):
                                         inspect_only=False,
                                         workflow_callback=workflow_callback,
                                         check_platforms=True)
-        assert 'Missing arches in manifest list' in str(exc_info.value)
+
+        base_image_with_registry = 'registry.example.com/{}'.format(BASE_IMAGE)
+        expected_msg = ('No images found for arches {} for base image {}'
+                        .format(sorted(missing_arches), base_image_with_registry))
```

Comment:
Rather than recalcating this in the same way the code under test does, could we include the expected output in the parametrization?

Reply:
While I was at it, I also changed the format of the output to not be a python list